services:
  people_tracking:
    start_tracking: "/butia_vision/pt/start"
    stop_tracking: "/butia_vision/pt/stop"
    
topics:
  people_tracking:
    people_detection: "/butia_vision/br/recognitions2D"
    people_tracking: "/butia_vision/pt/people_tracking"